data/scripting/field_animations.lua

```diff
@@ -84,7 +87,11 @@ function hide_randomly(plr, region, time)
    if delay < 1 then delay = 1 end
    while #region > 0 do
       local id = math.random(1, #region)
-      plr:hide_fields({region[id]},true)
+      if permanent then
+         plr:hide_fields({region[id]},true)
+      else
+         plr:unexplore_fields({region[id]})
+      end
```

Comment:
If you make the new switch the default then this affects all other occurrences of the function. Please make the old behaviour (`permanent=true`) the default, and use the new functionality only for emp03.

Reply:
Well, the hide randomly is only used in emp03. and the new function is exactly what was the default before the vision change from Niektory, so from my point of view I just reinstated the former default, which was changed by the vision code changes. 
Only instance of hide concentric is in emp01 which should be fine as well to be rexplorable.